tracker/data/bVNocbRzv.json

```diff
@@ -1 +1 @@
-{"id":"bVNocbRzv","document_name":"Test","peers":["288c036c-3e41-4049-80c4-31bdc047c1a3"],"data":[]}
\ No newline at end of file
+{"id":"bVNocbRzv","document_name":"Data","peers":[],"data":["Hello World!","Got data from tracker"]}
```

Comment:
I'm uninformed - what is the purpose of the files in this directory, and was there a reason the `peers` array was cleared?

Reply:
These are the two default sessions that exist in the tracker, and they show up when we open the homepage.

The peers array was cleared because it is supposed to save only existing peers that are currently in the session, and during startup no peers exist in that session.

Reply:
The tracker saves each session as a file once it’s created in the ui. These are loaded whenever the tracker restarts so we don’t lose any sessions.